catalog/USGS/USGS_NLCD_RELEASES_2021_REL_TCC_v2021-4.jsonnet

```diff
@@ -0,0 +1,390 @@
+local ee_const = import 'earthengine_const.libsonnet'; 
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local id = 'USGS/NLCD_RELEASES/2021_REL/TCC/v2021-4';
+local version = 'v2021-4';
+local subdir = 'USGS';
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+local license = spdx.proprietary;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'USFS Tree Canopy Cover ' + version + ' ' + '(CONUS and OCONUS)',
+  'gee:skip_indexing': true,
+  version: version,
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    This product is part of the Tree Canopy Cover (TCC) data suite. It includes modeled TCC, standard error (SE), and 
+    National Land Cover Database's (NLCD) TCC data for each year. TCC data produced by the the United States Department 
+    of Agriculture, Forest Service (USFS) are included in the Multi-Resolution Land Characteristics (MRLC) consortium 
+    that is part of the National Land Cover Database (NLCD) project managed by the United States (US) Geological Survey (USGS).
+
+    The Science TCC product and NLCD TCC are remote sensing-based map output produced by the USFS. The objecive of TCC Science 
+    and NLCD TCC are to develop a consistent approach using the latest technology and advancements in TCC mapping to produce a 
+    "best available" map of TCC across the Conterminous United States (CONUS) and southeast Alaska, Hawaii and 
+    Puerto Rico-US Virgin Islands (OCONUS).
+
+    Model outputs include Science TCC, Science SE and NLCD TCC. Science TCC include data 
+    from 2008 through 2021. Science TCC is the raw direct model outputs. Science SE is the model standard deviation 
+    of the predicted values from all regression trees. The NLCD TCC product undergoes further post processing applied 
+    to the annual Science TCC images, which includes several masking (water and non-tree agriculture), filtering, and 
+    minimum-mapping unit (MMU) routines, as well as processes that reduce interannual noise and return longer duration trends. 
+
+    For Science SE data, the initial SE estimates that ranged from 0 to approximately 45 were multiplied by 100 
+    to maintain data precision (e.g., 45 = 4500). Therefore, SE estimates pixel values range from 0 to approximately 4500. 
+    In initial model output the TCC value 254 and SE value 65534 represent the non-processing area mask where pixels within 
```

Comment:
Users will not see these values in the asset, right? I wouldn't bother mentioning them to avoid confusing people. I'd say something like

pixels within 
Comment
Users will not see these values in the asset, right? I wouldn't bother mentioning them to avoid confusing people. I'd say something shorter like
 
Areas with no cloud or cloud shadow-free data are masked out in the data band. The data mask band has the value of 2 for such non-processing areas.

Reply:
I have simplified this section to mirror your suggestions